frontend/package.json

```diff
@@ -67,11 +68,10 @@
   "overrides": {
     "@appbaseio/reactivesearch": {
       "react-redux": "^7.2.6"
-    }
-  },
-  "resolutions": {
+    },
     "nth-check": "^2.0.1",
-    "@adobe/css-tools": "4.3.1"
+    "@adobe/css-tools": "4.3.1",
```

Comment:
I thought we wanted to remove this from `package.json` after generating the integrity sha-id in `yarn.lock`

Reply:
I prefer to leave it to ensure that if we generate another yarn.lock it forces the non-vulnerable version of the package.